packages/reactotron-server/src/server/datastore/commands.ts

```diff
@@ -54,7 +54,20 @@ export class Commands {
   commands: Command[] = []
 
   addCommand(command: Command) {
-    this.commands.push(command)
+    this.commands.unshift(command)
+
+    if (this.commands.length > 300) { // Arbitrary, I know... but its a start somewhere
+      this.commands = this.commands.slice(0, 300)
```

Comment:
Sometimes magic numbers are required, but if they are, I usually like to define them in a named constant so their purpose is clear. Its not obvious from this line why or what 300 is / does

Reply:
I agree. I’d like this number to be configurable too. So maybe a constructor to pass it in? doesnt have to happen now though.

Reply:
Constructor pass in setup but didn't set up anything to actually pass it in. That part can come later.